include/zephyr/bluetooth/classic/avrcp.h

```diff
@@ -24,6 +24,13 @@ struct bt_avrcp_unit_info_rsp {
 	uint32_t company_id;
 };
 
+struct bt_avrcp_subunit_info_rsp {
+	uint8_t subunit_type;
+	uint8_t max_subunit_id;
```

Comment:
Missing documentation for all these structures fields. E.g. it's not clear that this member tells the user how many elements are in the next two arrays. Is this really the most appropriate name for this member? I had to look at the implementation to understand what it's for. Would e.g. `ext_subunit_count` be more descriptive? (and then rename the next two to use `ext_` instead of `extended_`).

Reply:
Hi @jhedberg ,

These field names, length, and arrangement are copied from the AV/C ducument directly. (TA Document 2001012, Figure 11.8)
There are <max_subunit_id> elements recorded in the next two arrays.

"ext_" is acceptble, I would change it later :)

Reply:
![image](https://github.com/user-attachments/assets/976e0423-2b19-4e45-8aa1-8e7d9c900cca)

Reply:
BTW. "extended_max_subunit_ID" is a typo from the original document, so I removed "max".

Reply:
@gzh-terry thanks for the spec reference. In that case it's fine, i.e. it's better to be aligned with the spec.